Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mongodm"
-version = "0.7.1"
+version = "0.7.2"
```

Comment:
Because of breaking change in prelude module:

```suggestion
version = "0.8.0"
```

Reply:
This won't be needed since the breaking prelude change was a mistake.